src/vue/components/parts/Power.vue

```diff
@@ -59,6 +59,23 @@ export default {
       return this.actor?.flags?.archmage?.diceFormulaMode ?? 'short';
     },
     powerDetailFields() {
+      const spellFields = game.settings.get("archmage", "secondEdition")
+        ? [
+          'spellLevel2',
+          'spellLevel3',
+          'spellLevel4',
+          'spellLevel5',
+          'spellLevel6',
+          'spellLevel7',
+          'spellLevel8',
+          'spellLevel9',
+        ]
```

Comment:
Uhm, isn't there a 10th level as well?

Reply:
Duh yeah you're right. I can fix that in a follow-up if you haven't already done it.